index.bs

```diff
@@ -127,7 +127,7 @@ or a combination of both.
 This specification relies on several other underlying specifications.
 
 : HTML5
-:: The concept of <dfn>origin</dfn> and the <dfn>Navigator/dfn> interface are defined in [[!HTML5]].
+:: The concept of <dfn for='web'>origin</dfn> and the <dfn>Navigator/dfn> interface are defined in [[!HTML5]].
```

Comment:
OK. Note I did not change this text at all, only the markup - this was required to disambiguate "origin" used in this context from the origin field in ClientData for Bikeshed.

If we need a better definition for origin, can we handle that in a separate issue?